aea/protocols/generator/common.py

```diff
@@ -322,6 +326,15 @@ def check_prerequisites() -> None:
             "Cannot find protocol buffer compiler! To install, please follow this link: https://developers.google.com/protocol-buffers/"
         )
 
+    result = subprocess.run(  # nosec
+        ["protoc", "--version"], stdout=subprocess.PIPE, check=True
+    )
+    result_str = result.stdout.decode("utf-8")
+    if LIBPROTOC_VERSION not in result_str:
+        raise FileNotFoundError(
+            f"Invalid version for protoc. Found: {result_str}. Required: {LIBPROTOC_VERSION}."
+        )
```

Comment:
Isn't it too strict to limit to protoc version 3.11.4? I originally created all of the generated protocols using 3.13. 

Maybe just check the major release version 3.x.x, or if there was a change in 3.11.4 required by our software, check that it is 3.11.4 or later.

Reply:
We don't have a choice atm. It should have been fixed ages ago as we essentially "fixed" it by our package hashing. We have a way to "unfix" it but that is a few releases away.